### DISCUSSION TITLE
Not abel to run scrreeni-py.exe on windows 11 machine

### DISCUSSION THREAD

**Discussion by eagles-eye-python** [pranjal-joshi/Screeni-py#155] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/99113018/215493355-f4913456-bb9d-4835-b953-354cc996dc79.png)
Hello when I am trying to run screeni-py.exe I am able to go till the screener part and when I select any screener its erroring out as shown the image link. can anyone help me in fixing this?

**Comment by pranjal-joshi** [comment, marked as answer]
Hi @eagles-eye-python 

HTTPS error suggests that Screeni-py is not able to reach the GitHub repo to fetch the data.

Make sure that your PC has internet connectivity.
Also ensure that there are no Proxy / Anti malware programs that are blocking the GitHub URL.
If the PC you are using belongs to any organization, It's recommended to use this tool on your personal computer to avoid such restrictions.

**Reply by eagles-eye-python** [reply]
Hello @pranjal-joshi , Thank you for your reply 
I am using my personal laptop and no proxy /antimalware programs are blocking Github URL.

**Reply by pranjal-joshi** [reply]
The error suggest an issue with HTTPS.. You may try testing it out with another internet service provider like your mobile hotspot and let me know.

**Reply by eagles-eye-python** [reply]
@pranjal-joshi , its working now on the same  ISP and I changed to mobile hotspot as well there also its working.

**Reply by eagles-eye-python** [reply]
@pranjal-joshi , sorry its only working in mobile hotspot now, again facing same issue with Broadband (Jio Fiber)

**Reply by pranjal-joshi** [reply]
It's unfortunate but you may contact your ISP regarding this.
That's all we can do!
